src/Data/CurrentUser/CurrentUserDataItem.ts

```diff
@@ -4,15 +4,22 @@ import { ensureString } from '../../utils/ensureString'
 import { isOptionalString } from '../../utils/isOptionalString'
 import { neoletterClient } from '../neoletterClient'
 import { pisaClient } from '../pisaClient'
+import { errorToast } from './errorToast'
 
 export const CurrentUser = provideDataItem('CurrentUser', {
   async get() {
     const user = await load(currentUser)
     if (!user) return {}
 
-    const neoletterProfile = await neoletterClient().get('my/profile')
-    if (!isNeoletterData(neoletterProfile)) {
-      throw new Error('Neoletter data is not in the expected format')
+    let neoletterProfile
+    try {
+      neoletterProfile = await neoletterClient().get('my/profile')
+      if (!isNeoletterData(neoletterProfile)) {
+        throw new Error('Neoletter data is not in the expected format')
```

Comment:
* [ ] Mentioning Neoletter may be confusing to visitors. How about something more neutral like "Invalid user profile" here and "Invalid user ID" for `whoami`?